website/docusaurus.config.js

```diff
@@ -160,6 +160,10 @@ module.exports = {
                         to: '/user_guide/activation_strategy',
                         from: '/user_guide/control_rollout',
                     },
+                    {
+                        from: '/advanced/impression_data',
```

Comment:
If we update the front end before merging the docs pr, we can do without the redirect, but it's probably no big deal either way.

Reply:
Let's keep the redirect here until we finish the redux refactoring.